content/courses/onchain-development/intro-to-anchor.md

```diff
@@ -353,6 +395,13 @@ pub struct InstructionAccounts {
 pub struct AccountStruct {
     data: u64
 }
+
+const DISCRIMINATOR: usize = 0;
+const DATA_SIZE: usize = 8;
+
+impl Space for AccountStruct {
+    const INIT_SPACE: usize = DISCRIMINATOR + DATA_SIZE;
+}
```

Comment:
Remove DATA_SIZE, use https://www.anchor-lang.com/docs/space#the-init-space-macro then space = DISCRIMINATOR + INIT_SPACE